### PR TITLE
Standardizing images in CSS

### PR DESCRIPTION
**Pull request by andrewconner**
@yipingliao @2is10 
- Removing the `../img` to be more consistent
- Removing quotes, because single quotes break `gulp-rev-all`. Will file a bug.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/common/services/keepWhoService.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ angular.module('kifi.keepWhoService', [])
         if (user && user.id && user.pictureName) {
           return '//djty7jcqog9qu.cloudfront.net/users/' + user.id + '/pics/' + width + '/' + user.pictureName;
         }
-        return '';
+        return '//www.kifi.com/assets/img/ghost.200.png';
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/common/services/keepWhoService.js:13]: okay to remove the domain?

**Comment by yipingliao** [comment]
Looks good!

File: kifi-backend/modules/shoebox/angular/src/common/services/keepWhoService.js (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/services/keepWhoService.js:13]: Nope :P

**Comment by 2is10** [comment]
:+1:

**Pull request merged**